articles/cosmos-db/request-units.md

```diff
@@ -32,11 +32,11 @@ To provide predictable performance, you need to reserve throughput in units of 1
 After reading this article, you'll be able to answer the following questions:  
 
 * What are request units and request charges?
-* How do I specify request unit capacity for a collection?
+* How do I specify request unit capacity for a container?
 * How do I estimate my application's request unit needs?
-* What happens if I exceed request unit capacity for a collection?
+* What happens if I exceed request unit capacity for a container?
 
-As Azure Cosmos DB is a multi-model database, it is important to note that this article refers to a collection/document for a document API, a graph/node for a graph API and a table/entity for the Table API. This article refers to concept of a collection, graph, or table as a container, and a a document, node, or entity as an item.
+As Azure Cosmos DB is a multi-model database, it is important to note that this article refers to a container/document for a document API, a graph/node for a graph API and a table/entity for the Table API. This article refers to concept of a container, graph, or table as a container, and a a document, node, or entity as an item.
```

Comment:
Actually, the two instances in this paragraph should be reversed as they are explaining the difference between an container and collection.